CI/integration_tests/calculators/test_green_kubo_self_diffusion_coefficients.py

```diff
@@ -61,4 +61,8 @@ def test_project(traj_file, true_values, tmp_path):
 
     computation = project.run.GreenKuboDiffusionCoefficients(plot=False)
 
+    for key, val in computation["NaCl"].data_dict.items():
```

Comment:
I usually use both just in case I need more information but here it is not important. This test I only changed as it needed to have some additional information removed from it. I can remove the data dict call though.